### PR TITLE
Fix spelling mistakes within the root of the directory

### PR DESCRIPTION
**Pull request by CleanMachine1**
By using codespell I have fixed all of the reported files within the root of the pi-apps repo

### CODE REVIEW

**Comment by theofficialgman** [comment]
I checked, all of these are mistakes and the corrections are correct.

**Comment by theofficialgman** [comment]
might be an interesting idea to add this as a github actions to always check the repo for spelling mistakes
https://github.com/codespell-project/actions-codespell

**Comment by CleanMachine1** [comment]
There is a lot of false positives so maybe not

File: api [hunk 1]
```diff
@@ -1456,7 +1456,7 @@ Check your internet connection and try again.")
   fi
   
   if grep -qF "fetch-pack: unexpected disconnect while reading sideband packet" <<<"$errors" ;then
-    error_caption+=("The git commend encountered this error: "\""fetch-pack: unexpected disconnect while reading sideband packet"\"" Check the stability of Internet connection and try again.
+    error_caption+=("The git commit encountered this error: "\""fetch-pack: unexpected disconnect while reading sideband packet"\"" Check the stability of Internet connection and try again.
```
Comment (CleanMachine1) [inline api:1459]: Is this meant to be command?

Comment (Botspot) [reply api:1459]: Yes.

Comment (CleanMachine1) [reply api:1459]:
```suggestion
    error_caption+=("The git command encountered this error: "\""fetch-pack: unexpected disconnect while reading sideband packet"\"" Check the stability of Internet connection and try again.
```

**Comment by Botspot** [comment]
There is a big gui update in the works that will update some of these files already. To avoid an unnecessary number of updates for our users, would it be possible to merge this PR to the `new-gui` branch?

**Comment by CleanMachine1** [comment]
Sure, just change the base, or we can wait for that and then merge from master

**Comment by theofficialgman** [comment]
rebased and branch changed to new-gui

File: createapp [hunk 2]
```diff
@@ -430,7 +430,7 @@ Still need help? Botspot can try to help you if you <a href="\""https://github.c
       output="$(yad "${yadflags[@]}" --form --on-top \
         --title="Uninstall script" --width=310 --height=400 \
         --text="Now it's time to make your uninstall script. This will be executed anytime somebody clicks the Uninstall button.
-A text editor should have openened... Never mind, you know what to do." \
```
Comment (Botspot) [inline createapp:433]: I can't understand how this one went unnoticed for so long. 😂

**Pull request merged**